src/controllers/baseHandler.ts

```diff
@@ -7,12 +7,14 @@ import { lowerCaseMessageCommands } from "../utils/lowerCaseMessageCommand";
 import { commandNotFound } from "./commandNotFound";
 import { helloCommand } from "./helloCommand";
 import { verifyCommand } from "./verifyCommand";
+import { mentionEachUser } from "./mentionEachUser";
 
 export async function baseHandler(
   message: discordMessageRequest,
   env: env
 ): Promise<JSONResponse> {
   const command = lowerCaseMessageCommands(message);
+
```

Comment:
options property of the message.data object is optional, so the following lines of code are unsafe:

//  assertion was added because the options type is optional also eslint had hard rules set so had to cross the border :)
// eslint-disable-next-line @typescript-eslint/no-non-null-assertion
displayType: message.data?.options![0]?.name,
// eslint-disable-next-line @typescript-eslint/no-non-null-assertion
options: message.data?.options![0]?.options![0]?.options,

To fix this bug, you should use the !. operator to unwrap the optional properties

Reply:
![image](https://github.com/Real-Dev-Squad/discord-slash-commands/assets/64726586/c995e462-9e69-4b3e-8950-858e4565aa8f)
I am using it right?

Reply:
!. operator only unwraps the first level of optional properties. By using the [0] index twice, we are able to unwrap the optional properties until we reach the desired value.